libs/getdns/Config.in

```diff
@@ -7,10 +7,16 @@ config GETDNS_ENABLE_STUB_ONLY
 		getdns can be configured for stub resolution mode only. (Removes libunbound dependency)
 	default y
 	
-config GETDNS_ENABLE_IDN
-	bool "GETDNS_ENABLE_IDN"
+config GETDNS_ENABLE_IDN_LIBIDN
+	bool "GETDNS_ENABLE_IDN_LIBIDN"
 	help 
 		getdns can be configured with some IDN Support. (Requires libidn dependency)
 	default n
+	
+config GETDNS_ENABLE_IDN_LIBIDN2
+	bool "GETDNS_ENABLE_IDN_LIBIDN2"
+	help 
+		getdns can be configured with some IDN Support. (Requires libidn2 dependency)
```

Comment:
It might be worth including here some comment on what the difference is?  Is one preferred?

Reply:
I believe one is just a newer version of the other with a few extra capabilities. However, I just noticed that Openwrt doesn't actually have libidn2 in the package repo. I'll submit an update to remove the libidn2 option.

From a getdns perspective, I believe both libraries support the same exact functionality. I think the devs were just giving users a choice. I actually have both turned off by default (to minimize dependencies).